<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Game of Life</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>741</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="QPushButton_import_2">
         <property name="text">
          <string>import file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QPushButton_export_2">
         <property name="text">
          <string>export file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QPushButton_start">
         <property name="text">
          <string>start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QPushButton_stop">
         <property name="text">
          <string>stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QPushButton_clean_up">
         <property name="text">
          <string>clean up</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="gameBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>501</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>Game of Life</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>100</y>
        <width>221</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="size">
         <property name="text">
          <string>Size of the Field:</string>
         </property>
         <property name="buddy">
          <cstring>spinBox_columns</cstring>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QSpinBox" name="spinBox_columns">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string> columns</string>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>x</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_row">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string> rows</string>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>200</y>
        <width>221</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Timer:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_timer">
         <property name="suffix">
          <string> seconds</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Kryptographie</string>
     </attribute>
     <widget class="QPushButton" name="pushButton2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>150</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>click</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMy_Menu">
    <property name="title">
     <string>My Menu</string>
    </property>
    <addaction name="actionHello_1"/>
    <addaction name="actionHello_2"/>
    <addaction name="separator"/>
    <addaction name="actionBye"/>
   </widget>
   <widget class="QMenu" name="menuBlo">
    <property name="title">
     <string>Blo\</string>
    </property>
    <addaction name="actionbubububu"/>
   </widget>
   <addaction name="menuMy_Menu"/>
   <addaction name="menuBlo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHello_1">
   <property name="text">
    <string>Hello 1</string>
   </property>
  </action>
  <action name="actionHello_2">
   <property name="text">
    <string>Hello 2</string>
   </property>
  </action>
  <action name="actionBye">
   <property name="text">
    <string>Bye</string>
   </property>
  </action>
  <action name="actionbubububu">
   <property name="text">
    <string>bubububu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
